src/bin/hexyl.rs

```diff
@@ -109,6 +109,19 @@ fn run() -> Result<(), AnyhowError> {
                      goes to an interactive terminal",
                 ),
         )
+        .arg(
+            Arg::new("color-mode")
+                .long("color-mode")
+                .takes_value(true)
+                .value_name("MODE")
+                .possible_values(&["auto", "16", "8-bit"])
+                .default_value("auto")
+                .env("HEXYL_COLOR_MODE")
+                .help(
+                    "The set of colors to use. The auto-mode currently is equivalent to 8-bit, \
+                     but this may change in the future",
```

Comment:
Should we document the `HEXYL_COLOR_MODE` variable or is that done by clap?

Maybe add a short explanation what "16" and "8-bit" means?